.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Preliminary Steps
+
+Please review these preliminary steps before logging your issue. You may find the information referenced may answer or explain the behaviour you are seeing. It'll help us to know you've reviewed this information.
```

Comment:
People have a tendency to gloss over these things, so it might help to move this below the checklist. The list does open with *"Please confirm you have..."* so it opens up nice and direct. One immediately sees that something is required first.

I'd also wrap this paragraph in an HTML comment too, so lazy reporters don't give us messier reports. 😉